<ui version="4.0" >
 <class>commchannelSelForm</class>
 <widget class="QDialog" name="commchannelSelForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Chennel Selection</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="Q3ButtonGroup" name="CanaliRadioGroup" >
     <property name="title" >
      <string>Channels</string>
     </property>
     <property name="exclusive" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="" >
      <property name="geometry" >
       <rect>
        <x>9</x>
        <y>25</y>
        <width>531</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="COM" >
           <property name="text" >
            <string>COM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Socket" >
           <property name="text" >
            <string>TCP/IP Socket</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="Q3Frame" name="frame4" >
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="Q3ButtonGroup" name="COMPortButtonGroup" >
              <property name="title" >
               <string>Com Ports</string>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <widget class="QRadioButton" name="rbCOM1" >
                 <property name="text" >
                  <string>COM 1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbCOM2" >
                 <property name="text" >
                  <string>COM 2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbCOM3" >
                 <property name="text" >
                  <string>ttyUSB0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbCOM4" >
                 <property name="text" >
                  <string>ttyUSB1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Q3ButtonGroup" name="BuadRatebuttonGroup" >
              <property name="title" >
               <string>BaudRate</string>
              </property>
              <property name="exclusive" >
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <widget class="QRadioButton" name="BR9600" >
                 <property name="text" >
                  <string>9600</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="BR38400" >
                 <property name="text" >
                  <string>38400</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="BR19200" >
                 <property name="text" >
                  <string>19200</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="BR115200" >
                 <property name="text" >
                  <string>115200</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Q3Frame" name="frame5" >
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="HostTextLabel" >
              <property name="text" >
               <string>HOST (ip address or name)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="host" />
            </item>
            <item>
             <widget class="QLabel" name="TCPPortTextLable" >
              <property name="text" >
               <string>TCP Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="port" />
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global" >qradiobutton.h</include>
  <include location="global" >qlineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
